src/calculation/DetailedResultsBuilder.ts

```diff
@@ -224,6 +261,48 @@ export function handlePopulationValues(
       popResult.result = false;
       popResult.observations = null;
     }
+  }
+  return populationResultsHandled;
+}
+
+export function handleStandardPopulationValues(
+  populationResults: PopulationResult[],
+  group: fhir4.MeasureGroup
+): PopulationResult[] {
+  // proportional/cohort handling
+  const populationResultsHandled = populationResults;
+  // Cannot be in all populations if not in IPP.
+  if (!getResult(PopulationType.IPP, populationResults)) {
```

Comment:
Yeah having a hard time with this. So if getResult() returns true, then the ! makes that false, right?

Reply:
Yes- this checks if the IPP result is false